### ISSUE TITLE
When I used "Stream the camera preview with the VideoKitCameraManager component", my Android package crashed

### ISSUE THREAD

**Issue by IronYasuo** [videokit-ai/videokit#157]
hello,I am the person who emailed you yesterday,Let me rephrase my problem here.
![image](https://github.com/user-attachments/assets/1bbabb7b-7824-4139-8635-3f4a37b21fe0)
Unity version:2022.3.17f1
android studio :
![image](https://github.com/user-attachments/assets/67fa53ed-9584-4bbf-8939-f50e37e7422c)
VideoKitCameraManager :
![image](https://github.com/user-attachments/assets/32aa4c60-c714-473c-b9b1-f10945d9096b)
![image](https://github.com/user-attachments/assets/ac66a958-f0da-4ae5-b5e6-658cd1ae749c)
My android device:
1.MI 10 (Android 10)
2.oppo reno9pro (Android 14)

**Comment by IronYasuo** [comment]
Supplement the error log
![HphibF941oormnx4bXacCGEnnuc](https://github.com/user-attachments/assets/89b885a1-4fa4-458a-a3d5-22c189918819)
![LRJxbfQvNoHeVtxMZuVcFKUnnZe](https://github.com/user-attachments/assets/2695139d-eef7-4914-8744-f41a10f6b7b9)
2024-11-13 16:19:03.590 14884-14978 m.bian.VideoKi          com.bian.VideoKit                    W  Accessing hidden method Lmiui/contentcatcher/sdk/ContentCatcherManager;->unregisterContentInjector(Lmiui/contentcatcher/sdk/Token;)V (greylist, linking, allowed)
2024-11-13 16:19:03.595 14884-15003 AudioManager            com.bian.VideoKit                    D  getStreamVolume isRestricted mode = 0
2024-11-13 16:19:03.995 14884-15073 NetworkSecurityConfig   com.bian.VideoKit                    D  No Network Security Config specified, using platform default
2024-11-13 16:19:04.421 14884-15098 AudioTrack              com.bian.VideoKit                    I  isLongTimeZoreData zoer date time 5 Seconds
2024-11-13 16:19:04.749 14884-15003 ACameraManager          com.bian.VideoKit                    I  427@getCameraIdList available number of camera is 2. 
2024-11-13 16:19:04.750 14884-15003 ACameraMetadata         com.bian.VideoKit                    E  filterDurations: malformed duration key 1638402! count 0, type 3
2024-11-13 16:19:04.750 14884-15003 ACameraMetadata         com.bian.VideoKit                    E  filterDurations: malformed duration key 1638403! count 0, type 3
2024-11-13 16:19:04.750 14884-15003 ACameraMetadata         com.bian.VideoKit                    E  filterDurations: malformed duration key 1638407! count 0, type 3
2024-11-13 16:19:04.750 14884-15003 ACameraMetadata         com.bian.VideoKit                    E  filterDurations: malformed duration key 1638408! count 0, type 3
2024-11-13 16:19:04.750 14884-15003 VideoKit                com.bian.VideoKit                    D  Camera2Device 0 preview resolution set to (1280,720)
2024-11-13 16:19:04.750 14884-15003 VideoKit                com.bian.VideoKit                    D  Camera2Device 0 photo resolution set to (6016,4512)
2024-11-13 16:19:04.750 14884-15003 VideoKit                com.bian.VideoKit                    D  Camera2Device 0 frame rate set to (10,30)
2024-11-13 16:19:04.751 14884-15003 ACameraMetadata         com.bian.VideoKit                    E  filterDurations: malformed duration key 1638402! count 0, type 3
2024-11-13 16:19:04.751 14884-15003 ACameraMetadata         com.bian.VideoKit                    E  filterDurations: malformed duration key 1638403! count 0, type 3
2024-11-13 16:19:04.751 14884-15003 ACameraMetadata         com.bian.VideoKit                    E  filterDurations: malformed duration key 1638407! count 0, type 3
2024-11-13 16:19:04.751 14884-15003 ACameraMetadata         com.bian.VideoKit                    E  filterDurations: malformed duration key 1638408! count 0, type 3
2024-11-13 16:19:04.751 14884-15003 VideoKit                com.bian.VideoKit                    D  Camera2Device 1 preview resolution set to (1280,720)
2024-11-13 16:19:04.751 14884-15003 VideoKit                com.bian.VideoKit                    D  Camera2Device 1 photo resolution set to (2592,1940)
2024-11-13 16:19:04.751 14884-15003 VideoKit                com.bian.VideoKit                    D  Camera2Device 1 frame rate set to (10,30)
2024-11-13 16:19:04.756 14884-15003 VideoKit                com.bian.VideoKit                    D  Camera2Device 1 preview resolution set to (1280,720)
2024-11-13 16:19:04.757 14884-15003 VideoKit                com.bian.VideoKit                    D  Camera2Device 1 frame rate set to (10,30)
2024-11-13 16:19:04.763 14884-15003 CRASH                   com.bian.VideoKit                    E  *** *** *** *** *** *** *** *** *** *** *** *** *** *** *** ***
2024-11-13 16:19:04.763 14884-15003 CRASH                   com.bian.VideoKit                    E  Version '2022.3.17f1 (4fc78088f837)', Build type 'Release', Scripting Backend 'il2cpp', CPU 'arm64-v8a'
2024-11-13 16:19:04.763 14884-15003 CRASH                   com.bian.VideoKit                    E  Build fingerprint: 'Xiaomi/umi/umi:10/QKQ1.191117.002/V12.0.8.0.QJBCNXM:user/release-keys'
2024-11-13 16:19:04.763 14884-15003 CRASH                   com.bian.VideoKit                    E  Revision: '0'
2024-11-13 16:19:04.763 14884-15003 CRASH                   com.bian.VideoKit                    E  ABI: 'arm64'
2024-11-13 16:19:04.763 14884-15003 CRASH                   com.bian.VideoKit                    E  Timestamp: 2024-11-13 16:19:04.763293937+0800
2024-11-13 16:19:04.763 14884-15003 CRASH                   com.bian.VideoKit                    E  pid: 14884, tid: 15003, name: UnityMain  >>> com.bian.VideoKit <<<
2024-11-13 16:19:04.763 14884-15003 CRASH                   com.bian.VideoKit                    E  uid: 10687
2024-11-13 16:19:04.763 14884-15003 CRASH                   com.bian.VideoKit                    E  signal 11 (SIGSEGV), code 1 (SEGV_MAPERR), fault addr --------
2024-11-13 16:19:04.763 14884-15003 CRASH                   com.bian.VideoKit                    E  Cause: null pointer dereference
2024-11-13 16:19:04.763 14884-15003 CRASH                   com.bian.VideoKit                    E      x0  0000000000000000  x1  0000000000000800  x2  ffffffffffffffc1  x3  0000000000000002
2024-11-13 16:19:04.763 14884-15003 CRASH                   com.bian.VideoKit                    E      x4  000000795bda3001  x5  0000000000000004  x6  0000000000008005  x7  44ff434864747068
2024-11-13 16:19:04.763 14884-15003 CRASH                   com.bian.VideoKit                    E      x8  000000795baaa07f  x9  000000000000006e  x10 0000000000000001  x11 0000000000000001
2024-11-13 16:19:04.763 14884-15003 CRASH                   com.bian.VideoKit                    E      x12 00000000090f2a60  x13 0555555555555555  x14 0000000000000006  x15 ffffffffffffffff
2024-11-13 16:19:04.763 14884-15003 CRASH                   com.bian.VideoKit                    E      x16 00000079edf568f0  x17 00000079edf481b0  x18 00000078d01366e8  x19 000000795c39b600
2024-11-13 16:19:04.763 14884-15003 CRASH                   com.bian.VideoKit                    E      x20 0000000000000800  x21 000000795bda2800  x22 0000007840639240  x23 000000795c3c6f40
2024-11-13 16:19:04.763 14884-15003 CRASH                   com.bian.VideoKit                    E      x24 000000795bda2800  x25 0000000000000800  x26 00000078efc6a248  x27 0000007903659020
2024-11-13 16:19:04.763 14884-15003 CRASH                   com.bian.VideoKit                    E      x28 000000795bb39000  x29 0000007903656770
2024-11-13 16:19:04.763 14884-15003 CRASH                   com.bian.VideoKit                    E      lr  000000795bb08b70  sp  0000007903656570  pc  000000795bb08ba4  pst 0000000060001000
2024-11-13 16:19:04.763 14884-15003 CRASH                   com.bian.VideoKit                    E  
                                                                                                    backtrace:
2024-11-13 16:19:04.763 14884-15003 CRASH                   com.bian.VideoKit                    E        #00 pc 00000000004a3ba4  <anonymous:000000795b665000>
2024-11-13 16:19:04.889 14884-15003 CRASH                   com.bian.VideoKit                    E  Tombstone written to: /storage/emulated/0/Android/data/com.bian.VideoKit/files/tombstone_00
2024-11-13 16:19:04.889 14884-15003 CRASH                   com.bian.VideoKit                    E  Forwarding signal 11
2024-11-13 16:19:04.889 14884-15003 libc                    com.bian.VideoKit                    A  Fatal signal 11 (SIGSEGV), code 1 (SEGV_MAPERR), fault addr 0x0 in tid 15003 (UnityMain), pid 14884 (m.bian.VideoKit)
2024-11-13 16:19:04.964 15318-15318 DEBUG                   pid-15318                            A  pid: 14884, tid: 15003, name: UnityMain  >>> com.bian.VideoKit <<<
2024-11-13 16:19:04.965 15318-15318 DEBUG                   pid-15318                            A        #00 pc 0000000000072ba4  /data/app/com.bian.VideoKit-aZ6bvLPhbC_n-LjniYAmiw==/lib/arm64/[libFunction.so](http://libfunction.so/) (FXNConfigurationGetUniqueID+116) (BuildId: 0db0da8fc75a900a61487dd169c0802c7c4ac13c)
2024-11-13 16:19:04.965 15318-15318 DEBUG                   pid-15318                            A        #01 pc 00000000014aecfc  /data/app/com.bian.VideoKit-aZ6bvLPhbC_n-LjniYAmiw==/lib/arm64/[libil2cpp.so](http://libil2cpp.so/) (BuildId: 4381b5b6bcf4ec99fda4a29fb13bdb53644a68f4)
2024-11-13 16:19:04.965 15318-15318 DEBUG                   pid-15318                            A        #02 pc 0000000000798588  /data/app/com.bian.VideoKit-aZ6bvLPhbC_n-LjniYAmiw==/lib/arm64/[libil2cpp.so](http://libil2cpp.so/) (BuildId: 4381b5b6bcf4ec99fda4a29fb13bdb53644a68f4)
2024-11-13 16:19:05.434 24669-15321 ActivityTaskManager     system_server                        W    Force finishing activity

**Comment by olokobayusuf** [comment]
This is caused by https://github.com/fxnai/fxn3d/issues/11 . Fix coming shortly.
